components/Ui/DataTable/UiDataTableSection.vue

```diff
@@ -1,6 +1,16 @@
 <template>
   <section>
-    <h2 v-text="sectionTitle" />
+    <div class="cds--row">
+      <div class="cds--col-md-5 cds--col-lg-8 cds--col-xlg-7 cds--col-max-6">
+        <h2 v-text="sectionTitle" />
+        <p>
+          Extension Events are intimate gatherings at various campuses, tailored
+          for a select few attendees. Due to their close-knit nature, we do not
+          provide links as these events primarily take place within the
+          institution's community.
+        </p>
```

Comment:
sorry @aitanaviudes - I just realized something...

This component (`UiDataTableSection`), is also being used on the /events/seminar-series page (https://qiskit.org/events/seminar-series)

you can see in the Preview branch for your PR, that including this paragraph here, is impacting that other page
[Seminar Series page preview](https://qiskit-org-pr-3550.dcq4xc5i083.us-south.codeengine.appdomain.cloud/events/seminar-series)

(don't worry, it's not impacting the live site yet)... 

to fix this, you can leverage the sectionDescription that you included below... and conditionalize this paragraph to _only_ show, if the `sectionDescription` prop is available... 

```suggestion
        <p v-if="sectionDescription">
          {{ sectionDescription }}
        </p>
```

Reply:
resolved in https://github.com/Qiskit/qiskit.org/pull/3550/commits/9e7d698cbf2a0f898bd132f2575590cc22107a32